### PR TITLE
[MM-24323] Fixing alignment issues on the mobile new channel sidebar

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
Fixing some alignment issues with the Phase 2 sidebar on mobile view.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-24323

#### Screenshots

Before:
image

After:
image

### CODE REVIEW

**Comment by asaadmahmood** [comment]
@devinbinnie Can I have before/after screenshots.

**Comment by devinbinnie** [comment]
@asaadmahmood Done :)

File: sass/responsive/_mobile.scss [hunk 1]
```diff
@@ -1586,9 +1590,13 @@
                 .SidebarChannel {
                     height: 45px;
 
-                    a,
-                    button {
+                    .SidebarLink {
                         height: 45px;
+                        align-items: center;
+
```
Comment (nevyangelova) [inline sass/responsive/_mobile.scss:1596]: Where is the display: flex? :D adding it should align vertically!

Comment (devinbinnie) [reply sass/responsive/_mobile.scss:1596]: https://github.com/mattermost/mattermost-webapp/blob/feature/experimental_sidebar_features-v2/sass/layout/_sidebar-left.scss#L893

Right here!

Comment (nevyangelova) [reply sass/responsive/_mobile.scss:1596]: Oh weird

**Comment by jgilliam17** [comment]
@devinbinnie 
I noticed mention badges are out of alignment in mobile view. Should I file separately or can this be addressed here in this PR?
Screen Shot 2020-07-01 at 9 38 37 AM

**Comment by devinbinnie** [comment]
> @devinbinnie
> I noticed mention badges are out of alignment in mobile view. Should I file separately or can this be addressed here in this PR?
> Screen Shot 2020-07-01 at 9 38 37 AM

Oh whoops, I'll fix it here. If you see any more issues let me know :)

**Comment by jgilliam17** [comment]
@devinbinnie oh, I think something broke
empty LHS
Screen Shot 2020-07-02 at 10 17 08 AM

**Comment by devinbinnie** [comment]
> @devinbinnie oh, I think something broke
> empty LHS
> Screen Shot 2020-07-02 at 10 17 08 AM

Looks like it's using the wrong server branch. I'll look into it.

**Comment by jgilliam17** [comment]
@devinbinnie still seeing empty LHS on the test server

**Comment by devinbinnie** [comment]
> @devinbinnie still seeing empty LHS on the test server

Yep I know, working on it :P there's some issues with the build flow. I'll let you know when it's ready :)

**Comment by jgilliam17** [comment]
Thank you @devinbinnie 
Mention badges are aligned, but no 3 dot menu when mentions are present in mobile view.
Is this expected from the UX standpoint? See gif. Thanks.
with mention in mobile view

**Comment by devinbinnie** [comment]
> Thank you @devinbinnie
> Mention badges are aligned, but no 3 dot menu when mentions are present in mobile view.
> Is this expected from the UX standpoint? See gif. Thanks.
> with mention in mobile view

@andrewbrown00 Is this something we want? On mobile devices there wouldn't be any hover state.

**Comment by andrewbrown00** [comment]
> > Thank you @devinbinnie
> > Mention badges are aligned, but no 3 dot menu when mentions are present in mobile view.
> > Is this expected from the UX standpoint? See gif. Thanks.
> > with mention in mobile view
> 
> @andrewbrown00 Is this something we want? On mobile devices there wouldn't be any hover state.

@devinbinnie yeah, will want it for mobile view for non-touch devices; using mobile view on the desktop.

**Comment by devinbinnie** [comment]
@jgilliam17 Ready for review.

**Pull request merged**